src/api_client.rs

```diff
@@ -459,8 +459,13 @@ impl Client {
         }
     }
 
-    pub async fn get_peer(peer_id: Multiaddr) -> Result<PeerDescription, Error> {
-        unimplemented!("")
+    pub async fn get_peer(&self, peer_id: PeerId) -> Result<PeerDescription, Error> {
+        let id = serde_json::to_string(&peer_id)
+            .unwrap()
+            .replace(&['\"'][..], "");
```

Comment:
I'd rather just impl `fmt::Display` in the upstream -- can you make a PR to do that and then refactor this?

Reply:
can add a `fmt::Display` impl here: https://github.com/ralexstokes/ethereum-consensus/blob/main/src/networking.rs#L24

and likely re-use the `b58` code from the `serde` module there

Reply:
ok - [this PR to eth_consensus ](https://github.com/ralexstokes/ethereum-consensus/pull/165) upstreams the display logic, and the API call is simplified.